lib/changelog.js

```diff
@@ -152,12 +153,12 @@ exports.methods.push({
             return callback(null);
         }
 
-        var repo = matches[2];
-        var v1 = matches[3];
-        var v2 = matches[4];
-        var newVersion = Semver.gt(v1, v2) ? v1 : v2;
+        const repo = matches[2];
```

Comment:
Just wondering, would it be more readable with destructuring ? (`const [,, repo, v1, v2] = matches`)